### PR TITLE
implement ExpIntegralEi and ExpIntegralE.

### PR DESCRIPTION
**Pull request by sn6uv**

### CODE REVIEW

File: mathics/builtin/specialfunctions.py [hunk 1]
```diff
@@ -858,3 +858,34 @@ def prepare_sympy(self, leaves):
...
+    <dl>
+    <dt>'ExpIntegralEi[$z$]'
+      <dd>returns the exponential integral function $Ei(z)$.
+    </dl>
+
+    >> ExpIntegralEi[2.0]
+     = 4.95423435600189016
+    '''
+
+    sympy_name = ''
```
Comment (bnjones) [inline mathics/builtin/specialfunctions.py]: Can we use sympy's `Ei` function here?

Comment (sn6uv) [inline mathics/builtin/specialfunctions.py]: Ah, yes. I was looking for something like that.

Comment (bnjones) [inline mathics/builtin/specialfunctions.py]: http://reference.wolfram.com/language/ref/ExpIntegralEi.html has some special values for `ExpIntegralEi[0]` and `ExpIntegralEi[Infinity]`, but it looks like SymPy doesn't automatically return them:
```
>>> Ei(0)
Ei(0)
>>> Ei(oo)
Ei(oo)
```
Maybe we could add a couple of rules for these?

Comment (sn6uv) [inline mathics/builtin/specialfunctions.py]: I've opened https://github.com/sympy/sympy/pull/11341 to add this functionality to sympy.

**Pull request merged**